### PR TITLE
Release/v0.1.3 #88

### PR DESCRIPTION
**Pull request by nesnoj**
New release scheduled for calendar week 38.
Discussion takes place in #88

See CHANGELOG

### CODE REVIEW

**Review by henhuy — APPROVED** [review]
Stemp app is working with release.
A "Go" from my side.

**Comment by nesnoj** [comment]
> Stemp app is working with release.
> A "Go" from my side.

Thx! :+1:
I'll also wait for @Bachibouzouk's review before merging..

File: doc/getting_started.rst
Comment (Bachibouzouk) [inline doc/getting_started.rst:18]: Cool addon :)

**Review by Bachibouzouk — APPROVED** [review]
I could not test the stemp_abw as it still had first install mistakes on my laptop. I read the changes and it seems good to me :) If @henhuy tested it and it worked, then I approve the PR as well !

Good job

**Comment by nesnoj** [comment]
Thx @Bachibouzouk !

> I could not test the stemp_abw as it still had first install mistakes on my laptop

At least you don't have DB dump which is necessary. Next week, the final release will be published which includes the data deployment via fixtures. If you want to restore the data locally before, you can find a dump here: .../05_Temp/Alexis/stemp_abw_db_dumps/stemp_abw_db_190910.backup

**Pull request merged**
